examples/ts/cli.ts

```diff
@@ -38,6 +39,7 @@ let [processPath, , exchangeId, methodName, ... params] = process.argv.filter (x
     , shouldCreateRequestReport = process.argv.includes ('--report')
     , shouldCreateResponseReport = process.argv.includes ('--response')
     , shouldCreateBoth = process.argv.includes ('--static')
+    , runTest = process.argv.includes ('--run-test')
```

Comment:
shouldn't this be triggered by run-tests? that way we could even trigger all the languages simultaneously